applications/nrf_desktop/configuration/nrf52833dk_nrf52833/prj_shell.conf

```diff
@@ -96,23 +95,9 @@ CONFIG_BT_HOGP_REPORTS_MAX=12
 CONFIG_ASSERT=y
 CONFIG_ASSERT_LEVEL=2
 
-CONFIG_LOG=y
-CONFIG_LOG_DEFAULT_LEVEL=2
-CONFIG_LOG_MODE_OVERFLOW=y
-CONFIG_LOG_PRINTK=y
-CONFIG_LOG_BUFFER_SIZE=4096
-CONFIG_LOG_PROCESS_THREAD_STACK_SIZE=1024
-
-CONFIG_CONSOLE=y
-CONFIG_USE_SEGGER_RTT=y
-CONFIG_SEGGER_RTT_BUFFER_SIZE_UP=4096
-CONFIG_RTT_CONSOLE=y
-CONFIG_UART_CONSOLE=n
+CONFIG_DESKTOP_LOG=y
 
-CONFIG_SHELL=y
-CONFIG_SHELL_BACKEND_SERIAL=n
-CONFIG_SHELL_BACKEND_RTT=y
-CONFIG_SHELL_PROMPT_RTT="nrf_desktop:~$ "
+CONFIG_DESKTOP_SHELL=y
```

Comment:
They would still be tested in the CI, but since enabling them is only matter of changing value of one Kconfig option, I think it's no longer necessary to keep them as separate configuration type. You can simply use test scenario that enables given Kconfig option apart from configuration defined in the given file.

Reply:
Can be done in a separate PR